ui/app/AppLayouts/Chat/panels/communities/CommunityPermissionsSettingsPanel.qml

```diff
@@ -0,0 +1,152 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls.chat 1.0
+
+import "../../layouts"
+
+SettingsPageLayout {
+    id: root
+
+    signal addPermission()
+
+    title: qsTr("Permissions")
+
+    content: Flickable {
+        contentWidth: mainLayout.width
+        contentHeight: mainLayout.height + mainLayout.anchors.topMargin
+        clip: true
+        flickableDirection: Flickable.AutoFlickIfNeeded
+
+        ColumnLayout {
+            id: mainLayout
+            anchors.top: parent.top
+            anchors.topMargin: Style.current.padding
```

Comment:
That one is strange. Padding for community settings content is already controlled by `SettingsPageLayout`. Why this one is different? Are designs inconsistent in that term (all setting pages should have the same padding for their content)? or does the `SettingsPageLayout` define wrong padding?

Reply:
> That one is strange. Padding for community settings content is already controlled by `SettingsPageLayout`. Why this one is different? Are designs inconsistent in that term (all setting pages should have the same padding for their content)? or does the `SettingsPageLayout` define wrong padding?

Yeah, there is inconsistency between some desings like the following, so it will be addressed in `SettingsPageLayout` by setting the padding to the biggest one (32) and asking @John-44 to confirm we can set the same margin for all `Community Settings` views!

![](https://cdn.discordapp.com/attachments/930843540718755850/985819448227282944/unknown.png)

![](https://cdn.discordapp.com/attachments/930843540718755850/985819586345713716/unknown.png)

Reply:
> > That one is strange. Padding for community settings content is already controlled by `SettingsPageLayout`. Why this one is different? Are designs inconsistent in that term (all setting pages should have the same padding for their content)? or does the `SettingsPageLayout` define wrong padding?
> 
> Yeah, there is inconsistency between some desings like the following, so it will be addressed in `SettingsPageLayout` by setting the padding to the biggest one (32) and asking @John-44 to confirm we can set the same margin for all `Community Settings` views!
> 
> ![](https://camo.githubusercontent.com/fe95fec8778c04830aa67239e9dc1a8431e135983c255eccb58f29ae8b4123b7/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3933303834333534303731383735353835302f3938353831393434383232373238323934342f756e6b6e6f776e2e706e67)
> 
> ![](https://camo.githubusercontent.com/baa147dccbb92a52a5710e46d1e51c5821efe3f84f71de893b4d75ec2eb918eb/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3933303834333534303731383735353835302f3938353831393538363334353731333731362f756e6b6e6f776e2e706e67)

@noeliaSD all should be 32px.  Yes we shouldn't follow inconsistencies in the designs, as the layouts are implemented if there are any inconsistencies in the designs these should be removed and we should make the layouts consistent.  Thanks for spotting this! :-)

Reply:
> 

Perfect! Already fixed in the base class `SettingsPageLayout`.